custom_components/tahoma/cover.py

```diff
@@ -183,23 +180,6 @@ def device_class(self):
             or DEVICE_CLASS_BLIND
         )
 
-    @property
-    def device_state_attributes(self):
```

Comment:
@tetienne could we add this as a breaking change? That the attribute name does change for those? 

I know we add RSSI also as attr['rssi] since this could be used across multiple Home Assistant integrations. I don't think that many people rely on `lock_originator` and `memorized_position`.

Reply:
If someone uses these attributes I want to meet him :) We can add them to the breaking change if you want. For the rssi, we report it has rssi level. I've checked in the other integration, most of the time it's just rssi. I can rename it, if you are ok.

Reply:
Hehe, I have no clue either why you should use them. 

Renaming rssi would make sense to me, to have it aligned to most other packages. Otherwise we could just have used the original CoreState property there.

Reply:
@tetienne RSSI is also not used that much to be honest. However, maybe we need to make sure the state names are aligned to the naming guidelines. I saw something like that in a recent PR.

`core:RSSILevelState` could be `core_rssi_level_state` or `rssi_level_state`

`core:NameState` could be `core_name_state` or `name_state`

Reply:
If you can find the PR you talk, it would be nice. We can change the attribute naming convention within another PR.

Reply:
https://github.com/home-assistant/core/pull/38138#discussion_r460442412